src/main/java/core/basesyntax/util/CsvUtils.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.util;
+
+import static core.basesyntax.transactions.FruitsTransaction.Operation;
+
+import core.basesyntax.transactions.FruitsTransaction;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvUtils {
+    private static final String COMMA_DELIMETER = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    public static List<FruitsTransaction> readTransactions(Path path) {
+        List<FruitsTransaction> transactions = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(path.toFile()))) {
+            String line = reader.readLine();
+            while ((line = reader.readLine()) != null) {
+                line = line.trim();
+                if (line.isEmpty()) {
+                    continue;
+                }
+
+                String[] tokens = line.split(COMMA_DELIMETER);
+                if (tokens.length < 3) {
+                    System.err.println("Invalid line format: " + line);
+                    continue;
+                }
+
+                try {
```

Comment:
again, SRP violation - reading and parsing should be separate, Reader class isn't bothered with the parsing, business logic, it just reads from the filepath, parser class shouldn't be concerned with how did we get this data, he just forms custom objects from string or list of strings